src/test/java/com/google/api/codegen/testdata/nodejs_doc_main_library.baseline

```diff
@@ -115,13 +115,42 @@ function LibraryServiceClient(gaxGrpc, grpcClients, opts) {
 
   var bundleDescriptors = {
     publishSeries: new gax.BundleDescriptor(
-        'books',
-        [
-          'edition',
-          'shelf.name'
-        ],
-        'bookNames',
-        gax.createByteLengthFunction(grpcClients.libraryServiceClient.google.example.library.v1.Book))
+      'books',
+      [
+        'edition',
+        'shelf.name'
+      ],
+      'bookNames',
+      gax.createByteLengthFunction(grpcClients.libraryServiceClient.google.example.library.v1.Book))
+  };
+
+  var operationsClient = new gax.lro({
+    servicePath: servicePath,
+    port: port,
+    sslCreds: sslCreds,
+    clientConfig: clientConfig,
+    appName: appName,
+    appVersion: appVersion
+  });
+
+  var protoDescriptorPool = {};
+
+  extend(protoDescriptorPool, gaxGrpc.load([{
+    root: require('google-proto-files')('..'),
+    file: 'library.proto'
+  }]));
+
+  extend(protoDescriptorPool, gaxGrpc.load([{
+    root: require('google-proto-files')('..'),
+    file: 'tagger.proto'
+  }]));
```

Comment:
I'm afraid that this does not work. `extend` overrides existing properties on the new items, and because `gaxGrpc.load` returns `google` as for its top namespace, the entire protoDescriptorTool result will be replaced by the result of `tagger.proto`.

See:

```javascript
> var extend = require('extend');
undefined
> var x = {foo: {bar: {baz: 1}}}
undefined
> var y = {foo: {baz: 2}}
undefined
> extend(extend({}, x), y)
{ foo: { baz: 2 } }
```

You should use `lodash.merge` if you need things like this.